### PR TITLE
2021q4: add lldb report

### PR DESCRIPTION
**Pull request by mgorny**
CC @krytarowski, @emaste

### CODE REVIEW

File: 2021q4/lldb.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+by implementing a fully-featured serial port support and then support
+for FreeBSD kernel core dumps (vmcores).
+
+The LLDB client gained much improved support for connecting to the remote
+server over the serial port, and LLDB server gained support for accepting
+communication over the serial port.  This opened the possibility of using
+LLDB to debug embedded devices that use the RS232 interface.  It can also aid
+debugging kernels on regular PCs as it does not rely on the network stack.
+
+The support for FreeBSD vmcores has also been added to LLDB.  This makes it
```
Comment (pauamma) [inline 2021q4/lldb.adoc:34]: s/The support/Support/

File: 2021q4/lldb.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+replacement.  This project spans from July 2021 to January 2022 and aims to
+make LLDB suitable for debugging FreeBSD kernels.
+
+The earlier part of the project was focused on improving compatibility between
+LLDB and other servers implementing the GDB Remote Protocol.  This was followed
+by implementing a fully-featured serial port support and then support
+for FreeBSD kernel core dumps (vmcores).
+
+The LLDB client gained much improved support for connecting to the remote
+server over the serial port, and LLDB server gained support for accepting
```
Comment (pauamma) [inline 2021q4/lldb.adoc:29]: "a serial port" and "the LLDB server", I think.

File: 2021q4/lldb.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+communication over the serial port.  This opened the possibility of using
```
Comment (pauamma) [inline 2021q4/lldb.adoc:30]: "a serial port", as above.

**Review by pauamma — CHANGES_REQUESTED** [review]
LGTM (unless I missed something) other than that.

**Comment by mgorny** [comment]
Thanks. English is really hard, let's write in Esperanto the next quarter ;-).

**Review by pauamma — APPROVED** [review]
LGTM now.

**Pull request merged**